Orange/widgets/model/owneuralnetwork.py

```diff
@@ -222,6 +222,7 @@ def _task_finished(self, f):
         else:
             self.model.name = self.learner_name
             self.model.instances = self.data
+            self.model.skl_model.orange_callback = None  # remove unpicklable callback
```

Comment:
This fixes the widget output which can now be pickled. But a general NN model (unfixed, not produced by the widget) is still unpicklable right? Are we fine with that?